### ISSUE TITLE
Deleted War-room has returned and cannot be deleted now.

### ISSUE THREAD

**Issue by ilyam8** [netdata/netdata-cloud#93]
@bazwalt commented on [Wed May 12 2021](https://github.com/netdata/netdata/issues/11135)

Hi all,

I have a war-room in my space that was deleted yesterday evening and it has subsequently returned this morning upon my login.

I now cannot delete it again and it keeps telling me I am no longer a room member. I also cannot re-add myself to the room.

**Comment by dimko** [comment]
@jacekkolasa any updated on this one?

**Comment by ivangergo3** [comment]
@vkuznecovas do you think it can be caused by the DB inconsistency?

**Comment by vkuznecovas** [comment]
Yes, but without more info it's hard to find a definite answer. Space IDs, roomIDs would help.

**Comment by ivangergo3** [comment]
@ilyam8 Could you please provide us with the Space ID or room ID?

**Comment by ilyam8** [comment]
@ivangergo3 I am not the author, I [moved the issue](https://github.com/netdata/netdata/issues/11135#issuecomment-839556635) from netdata/netdata via ZenHub.

@bazwalt are you still having the problem? If so, can you answer @ivangergo3 question?

**Comment by cakrit** [comment]
We know where this issue is coming from, desynchronization between the materialized views and the DB. There's active work to resolve such issues permanently, but it will take a while. We don't even have the ability to manually correct such discrepancies, so there's unfortunately nothing that can be done for individual cases where this happens. It was an old bad design decision that will take some time to undo, but we have people working full time on doing exactly that.
